Src/ElasticScale.Client/ShardManagement/ShardMap/ShardMapUtils.cs

```diff
@@ -71,5 +92,27 @@ static ShardMapUtils()
                             CultureInfo.InvariantCulture);
             }
         }
+
+        /// <summary>
+        /// Validate .Net runtime 4.6 if Active Directory Aughentication is specified in connection string.
+        /// </summary>
+        /// <param name="connectionString">Connection string to validate.</param>
+        internal static void ValidateAuthenticationInConnectionString(string connectionString)
```

Comment:
We are basically just replacing the SqlClient error type & message with our own, right? I think this is not needed. The SqlClient error message is already localized and googleable.